src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -35,6 +36,7 @@ type ActionCellProps = {
 
 function ActionCell({
     action = CONST.SEARCH.ACTION_TYPES.VIEW,
+    isOnHold,
```

Comment:
I don't think that `ActionCell` should be aware of this. I think we should pass a `shouldUseSuccessStyle` prop like we're doing for the settlement button.

Reply:
Agreed.